clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -6,10 +6,11 @@
 - Allow viewing content generated by AINSERT instruction and preprocessors
 - Expand the list of associated file extensions
 - DB2 preprocessor now supports the VERSION option
-- Instruction set versioning support
+- Instruction set versioning support (via OPTABLE and MACHINE options)
 - Basic GOFF, XOBJECT and SYSOPT_XOBJECT support
 - MNOTE support
 - Assembler options can be specified in pgm_conf.json to override values from proc_grps.json
+- User is now prevented from entering synonymous assembler option entries
```

Comment:
I don't think we need the second entry, this is a change to an unrelease feature.

Reply:
Ok